app/src/main/java/com/example/grassroots/network/CivicInfoPresenter.java

```diff
@@ -0,0 +1,26 @@
+package com.example.grassroots.network;
+
+import android.content.Context;
+
+import com.example.grassroots.fragment.RepresentativeDirectoryFragment;
+import com.example.grassroots.model.CivicInfoModel;
+
+public class CivicInfoPresenter {
+
+    private RepresentativeDirectoryFragment representativeDirectoryFragment;
+
+    public CivicInfoPresenter(RepresentativeDirectoryFragment representativeDirectoryFragment) {
```

Comment:
Presenters should not take in a fragment. This would usually be done with an interface.